_locales/_untranslated_en.json

```diff
@@ -11,5 +11,14 @@
   },
   "background_sync_disabled_explaination": {
     "message": "Background Sync Disabled, Account is only synced when selected"
+  },
+  "account_muted_description": {
+    "message": "Notifications are disabled for this Account"
+  },
+  "pref_for_all_accounts": {
+    "message": "For All Accounts"
```

Comment:
The uppercase is a little confusing, I'd write "For all accounts" and "For the current account" or even better: "All accounts" and "Current account"

Reply:
we have this strange casing for the other heading in in the settings too.

Reply:
the "strange casing" was pushed not least from the former desktop team at https://github.com/deltachat/deltachat-android/pull/1979 :)

titles may be a corner case, but i think it is good to use the "mostly upper" also there, i also still agree with the former decision, it still matches nicer in general to most other apps and conventions.

but let's go for for "All Account"/"Current Account" then, if it is really needed to mention account scope (it is an information that may open questions to the remaining settings dialog, i'd like to get rid of that, but i do not have a good alternative yet)

Reply:
> the "strange casing" was pushed not least from the former desktop team at https://github.com/deltachat/deltachat-android/pull/1979 :)

I thought it was the people writing the website, not sure who came up with this first xD